dht22_exporter.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 
 # Prometheus exporter for DHT22 running on raspberrypi
-# Usage: ./dht22_exporter -g <gpio_pin_number> -i <poll_time_in_seconds> [-p <exposed_port>]
+# Usage: ./dht22_exporter -g <gpio_pin_number> -i <poll_time_in_seconds> [-a <exposed_address> -p <exposed_port>]
```

Comment:
Not that I wrote this as a compliant usage string in the first place, but I think usage strings are normally bracketed by themselves. So this would be `[-a <exposed_address>] [-p <exposed_port>]`.